org.zrythm.Zrythm.json

```diff
@@ -0,0 +1,442 @@
+{
+    "app-id": "org.zrythm.Zrythm",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "zrythm_launch",
+    "rename-icon": "zrythm",
+    "copy-icon": true,
+    "finish-args": [
+        "--share=ipc",
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        
+        "--share=network",
```

Comment:
does it really require network?

Reply:
it's only used for checking for new versions at the moment

Reply:
```suggestion
```

Reply:
actually this is used for sending bug reports automatically too. preferably the flatpak should include debug symbols for proper backtraces, I think @vchernin is looking into that